weblogic-azure-aks/src/main/arm/scripts/inline-scripts/enablePrometheusMetrics.sh

```diff
@@ -0,0 +1,411 @@
+# Copyright (c) 2024, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+function connect_aks(){
+    az aks get-credentials \
+        -n $AKS_CLUSTER_NAME \
+        -g $AKS_CLUSTER_RG_NAME \
+        --overwrite-existing \
+        --only-show-errors
+}
```

Comment:
The following scripts seem to all have a need for this same functionality. If this is common functionality, why then is it not in `utility.sh`? There needs to be a good reason for this DRY violation.

```
az aks get-credentials \
./weblogic-azure-aks/src/main/arm/scripts/inline-scripts/queryDomainConfigurations.sh
    az aks get-credentials \
./weblogic-azure-aks/src/main/arm/scripts/inline-scripts/validateAgic.sh
    az aks get-credentials \
./weblogic-azure-aks/src/main/arm/scripts/inline-scripts/enableHpa.sh
    az aks get-credentials \
./weblogic-azure-aks/src/main/arm/scripts/inline-scripts/enablePrometheusMetrics.sh
    az aks get-credentials \
./weblogic-azure-aks/src/main/arm/scripts/updateApplications.sh
    az aks get-credentials \
./weblogic-azure-aks/src/main/arm/scripts/validateApplications.sh
    az aks get-credentials --resource-group ${AKS_CLUSTER_RESOURCEGROUP_NAME} --name ${AKS_CLUSTER_NAME} --overwrite-existing
./weblogic-azure-aks/src/main/arm/scripts/setupWLSDomain.sh
  az aks get-credentials \
./weblogic-azure-aks/src/main/arm/scripts/queryStorageAccount.sh
    az aks get-credentials \
./weblogic-azure-aks/src/main/arm/scripts/setupDBConnections.sh
  az aks get-credentials --resource-group ${AKS_CLUSTER_RG_NAME} --name ${AKS_CLUSTER_NAME} --overwrite-existing
./weblogic-azure-aks/src/main/arm/scripts/setupNetworking.sh
```

Reply:
See https://github.com/oracle/weblogic-azure/pull/297/commits/d6cb492769ef50fffcb502cd3d6c11ff741fd7e8

Reply:
Thanks. I'll mark this as resolved.